media/js/newsletter/mozilla.js

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+$(function () {
+    'use strict';
+
+    $(document).ajaxSuccess(function(evt, xhr, settings, response) {
```

Comment:
Just to make things really obvious, add a comment here explaining what this is doing?